core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -610,6 +610,22 @@ static bool bn_alloc_max(struct bignum **s)
 	return !!(*s);
 }
 
+static TEE_Result __maybe_unused convert_ltc_verify_status(int ltc_res,
+							   int ltc_stat)
+{
+	switch (ltc_res) {
+	case CRYPT_OK:
+		if (ltc_stat == 1)
+			return TEE_SUCCESS;
+		else
+			return TEE_ERROR_SIGNATURE_INVALID;
+	case CRYPT_INVALID_PACKET:
```

Comment:
why this specific conversion?

Reply:
Because of https://github.com/OP-TEE/optee_os/pull/1732#discussion_r131431450.
If the signature can't be parsed, it is invalid.